app/assets/javascripts/select-kit/addon/components/category-selector.js

```diff
@@ -113,14 +114,13 @@ export default MultiSelectComponent.extend({
         makeArray(this.selectedContent).concat(item.category.descendants)
       );
     } else {
-      this._super(value, item);
+      super.select(value, item);
     }
-  },
-
-  actions: {
-    onChange(values) {
-      this.onChange(values.map((v) => Category.findById(v)).filter(Boolean));
-      return false;
-    },
-  },
-});
+  }
+
+  @action
+  _onChange(values) {
```

Comment:
Renamed to avoid clash with the `@onChange` argument. This works because select-kit will call the underscored version of actions first.